### DISCUSSION TITLE
投票：删除行列后是否减少行列数

### DISCUSSION THREAD

**Discussion by dream-num** [dream-num/Luckysheet#431] [Ideas]
**问题：** 如何删除一行，并改变总行的序号，不只是删除内容。比如：现在我删除最后没有内容的一行，给人的感觉是没有删掉。

**解决方案：** 参照腾讯文档/google sheet，删除行或者列，仍然保持顺序补全，但是行数或者列数需要变化才对

是否支持该方案？支持请点赞，根据点赞数来决定是否需要用该方案

**Comment by fzzflxk** [comment]
我觉得还是有这个需要的，我自己就遇到这个需求，不知道大家遇到这个需求了没有。 还有不知道测试小姐姐会不会给我提BUG 。。

**Reply by ZCF1024** [reply]
这种删除感觉还是挺合理的，如果设计一个删除动效给人一种删除的体验可能会更好些。

**Reply by fzzflxk** [reply]
可以新增一个方法，原 deleteRow 方法保留。

**Reply by simonwater** [reply]
同样有这个需求

**Reply by Gourdbaby** [reply]
不想只删除内容，删除内容有删除内容的用处，删除行 就应该删除整行，不应该还保留在当前表格中，要不就不叫删除行，叫删除内容。
